pkgs/tools/misc/ncdu/default.nix

```diff
@@ -19,6 +19,9 @@ stdenv.mkDerivation rec {
 
   PREFIX = placeholder "out";
 
+  # Avoid CPU feature impurity, see https://github.com/NixOS/nixpkgs/issues/169461
+  ZIG_FLAGS = "-Drelease-safe -Dcpu=baseline";
```

Comment:
Why is release-safe required, instead of just setting the CPU to baseline?

Reply:
It's probably not required, but building with a "release" [mode](https://ziglang.org/documentation/master/#Build-Mode) seems reasonable for nixpkgs (mostly for reproducible builds), and `release-safe` is the one that's used [in most other places](https://github.com/NixOS/nixpkgs/issues/185665#issuecomment-1240555770) so I went with that one.

Which release mode to enable is a discussion that should be had when the zig setup hook is created.